smart_control/simulator/building.py

```diff
@@ -757,8 +837,116 @@ def __init__(
     self.neighbors = self._calculate_neighbors()
     self.len_neighbors = self._calculate_length_of_neighbors()
 
+    self._assign_radiative_heat_transfer_properties(
+        view_factor_method,
+        exterior_walls,
+        interior_walls,
+        inside_wall_radiative_properties,
+        building_exterior_radiative_properties,
+        inside_air_radiative_properties,
+    )
+
     self.reset()
 
+  def _assign_radiative_heat_transfer_properties(
+      self,
+      view_factor_method,
+      exterior_walls,
+      interior_walls,
+      inside_wall_radiative_properties,
+      building_exterior_radiative_properties,
+      inside_air_radiative_properties,
+  ):
+    if self.include_radiative_heat_transfer:
+      self.view_factor_method = view_factor_method
+
+      self.indexed_floor_plan = self.floor_plan.copy()
+      # 2=>-1
+      self.indexed_floor_plan[
+          self.indexed_floor_plan
+          == constants.EXTERIOR_SPACE_VALUE_IN_FILE_INPUT
+      ] = constants.EXTERIOR_SPACE_VALUE_IN_FUNCTION
+      # 1 to -3
+      self.indexed_floor_plan[
+          self.indexed_floor_plan == constants.INTERIOR_WALL_VALUE_IN_FILE_INPUT
+      ] = constants.INTERIOR_WALL_VALUE_IN_FUNCTION
+
+      self.interior_wall_mask = (
+          building_radiation_utils.mark_interior_wall_adjacent_to_air(
+              self.indexed_floor_plan,
+              constants.INTERIOR_WALL_VALUE_IN_FUNCTION,
+              constants.INTERIOR_SPACE_VALUE_IN_FUNCTION,
+          )
+      )
+      self.interior_wall_index = np.full(self.indexed_floor_plan.shape, -1)
+      self.interior_wall_index[self.interior_wall_mask] = np.arange(
+          np.sum(self.interior_wall_mask)
+      )
+      self.interior_wall_VF = building_radiation_utils.get_VF(  # pylint: disable=invalid-name
+          self.indexed_floor_plan,
+          self.interior_wall_mask,
+          constants.INTERIOR_WALL_VALUE_IN_FUNCTION,
+          view_factor_method=view_factor_method,
+      )
+
+      # radiative properties
+      # by default, all radiative properties are 0.0
+      inside_wall_radiative_properties = (
+          inside_wall_radiative_properties
+          or RadiationProperties(epsilon=0.0, alpha=0.0, tau=0.0)
```

Comment:
Instead of defaulting these properties to zero, we would like to be able to specify some "reasonable" defaults for epsilon, alpha, and tau for the inside wall properties, exterior properties, and inside air properties.

It would be nice to be able to say that, for example, "a glass wall would be equivalent to values X, Y, and Z".  

Is there a place we can look these up? It would be nice to link to more information, and perhaps include a few different options in a lookup table in the code and/or docs.